### PR TITLE
Issue 663 - Docker Launch Overrides

### PR DESCRIPTION
**Pull request by andresrinivasan**
Re: #663.

I found having a mix of Docker based microservices and local microservices useful while developing.

### CODE REVIEW

File: bin/edgex-docker-launch.sh [hunk 1]
```diff
@@ -7,16 +7,37 @@
...
+# EDGEX_SERVICES lists the service to start
+#
+# E.g. Start everything but metadata and data, use Redis for Core Services and a local compose file
+# EDGEX_SERVICES="logging command export-client export-distro notifications" \
+# EDGEX_CORE_DB=redis EDGEX_COMPOSE_FILE=docker/local-docker-compose.yml \
+# bin/edgex-docker-launch.sh
+
+if [[ -z $EDGEX_COMPOSE_FILE ]]; then
+  COMPOSE_FILE=$GOPATH/src/github.com/edgexfoundry/edgex-go/docker/docker-compose.yml
+  COMPOSE_URL=https://raw.githubusercontent.com/edgexfoundry/developer-scripts/master/compose-files/docker-compose-california-0.6.0.yml
```
Comment (tsconn23) [inline bin/edgex-docker-launch.sh]: COMPOSE_URL is out of date. Should point to Delhi. See previous version.

File: bin/edgex-docker-launch.sh [hunk 2]
```diff
@@ -7,16 +7,37 @@
...
+# EDGEX_CORE_DB identifies the DB for Core Services.
+# EDGEX_SERVICES lists the service to start
+#
+# E.g. Start everything but metadata and data, use Redis for Core Services and a local compose file
+# EDGEX_SERVICES="logging command export-client export-distro notifications" \
+# EDGEX_CORE_DB=redis EDGEX_COMPOSE_FILE=docker/local-docker-compose.yml \
+# bin/edgex-docker-launch.sh
+
+if [[ -z $EDGEX_COMPOSE_FILE ]]; then
+  COMPOSE_FILE=$GOPATH/src/github.com/edgexfoundry/edgex-go/docker/docker-compose.yml
```
Comment (tsconn23) [inline bin/edgex-docker-launch.sh]: With modules coming, project location may not necessarily be in the $GOPATH. Please revert to previous relative path.

File: bin/edgex-docker-launch.sh (hunk 1 above)
Comment (andresrinivasan) [reply bin/edgex-docker-launch.sh]: Done

File: bin/edgex-docker-launch.sh (hunk 2 above)
Comment (andresrinivasan) [reply bin/edgex-docker-launch.sh]: Done

**Pull request merged**